### PR TITLE
Add zoom example

### LINKED ISSUE #103: Replace redEye example with one exercising zoom.
Replace example num.2 https://w3c.github.io/mediacapture-image/#taking-a-picture-with-red-eye-reduction-supported-and-used with an example manipulating e.g. Zoom, that is more visual and is already available, while the changes in #73 are being addressed. Will do that quickly.

### PR DESCRIPTION
**Pull request by beaufortfrancois**
FIX #103

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -500,33 +500,36 @@
...
+        input.value = photoCapabilities.zoom.current;
+        input.oninput = function(event) {
+          imageCapture.setOptions({zoom: event.target.value});
+        }
+      });
     }
 
     function failedToGetMedia(e) {
-        console.log('Stream failure: ' + e);
+      console.log('Stream failure: ' + e);
```
Comment (miguelao) [inline index.html]: console.error(...)?

File: index.html [hunk 2]
```diff
@@ -500,33 +500,36 @@
     }
     </pre>
 
-    ##### Taking a picture with Red Eye Reduction supported and used
+    ##### Update device zoom
     <pre class='example'>
-    navigator.mediaDevices.getUserMedia({video: true}).then(gotMedia, failedToGetMedia);
+    var imageCapture;
+
+    navigator.mediaDevices.getUserMedia({video: true}).then(gotMedia).catch(failedToGetMedia);
```
Comment (miguelao) [inline index.html]: The changes here lgtm.  I was wondering if we could upload this examples in some kind of codepen/jsfiddle or similar, and link (informatively) from these examples, so the casual spec reader can just click through and see these wonderful demos in action :smile: -- my question would be if w3c has some specific way of doing it.

**Comment by beaufortfrancois** [comment]
PR updated.

**Comment by miguelao** [comment]
@beaufortfrancois it seems like this pull has a merge conflict, could you please address it? Thanks!

**Comment by beaufortfrancois** [comment]
Merge conflict is addressed.

**Comment by beaufortfrancois** [comment]
I'm not sure why https://labs.w3.org/hatchery/repo-manager/pr/id/w3c/mediacapture-image/111 fails though ;(

**Pull request merged**